fineract-investor/src/main/java/org/apache/fineract/investor/api/ExternalAssetOwnersApiResourceSwagger.java

```diff
@@ -152,4 +152,57 @@ static final class ExternalAssetOwnerTransferChangesData {
             public String purchasePriceRatio;
         }
     }
+
+    @Schema(description = "GetExternalAssetOwnerSearchRequest")
+    public static final class GetExternalAssetOwnerSearchRequest {
+
+        private GetExternalAssetOwnerSearchRequest() {}
+
+        @Schema(example = "2023-05-23")
+        public String settlementFromDate;
+
+        @Schema(example = "2023-06-23")
+        public String settlementToDate;
+
+        @Schema(example = "2023-05-23")
+        public String effectiveFromDate;
+
+        @Schema(example = "2023-06-23")
+        public String effectiveToDate;
+
+        @Schema(example = "1234567890987654321abc")
+        public String text;
+    }
+
+    @Schema(description = "GetSearchExternalAssetOwnerTransferResponse")
```

Comment:
We dont need this. Swagger can automatically generate this. See `ExternalAssetOwnersApiResource.getTransfers`